lib/Game.js

```diff
@@ -21,14 +21,18 @@ class Game {
     const puzzleWordArray = this.currentAnswer.split(' ');
     this.answerLettersArray = puzzleWordArray.map((word) => {
       return word.split('');
-    })
+    });
   };
 
   checkDelimiters(character, positionNum) {
     if (character === '-') {
       $(`.letter-card-div-${positionNum}`).children().removeClass('hidden');
       $(`.letter-card-div-${positionNum}`).css({"background-color": "#27A067"});
     }
+    if (character === '\'') {
+      $(`.letter-card-div-${positionNum}`).children().removeClass('hidden');
+      $(`.letter-card-div-${positionNum}`).css({"background-color": "#27A067"});
```

Comment:
This is great! a good solution to solve the problem of our left over puzzle tiles looking like they are on the board.